### PR TITLE
Recommend

### PR DESCRIPTION
**Pull request by bbfsdev**
- Recommend block per topic and per collection.
- Limit popular to be more than 100 views.
- Update recommendation labels.

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -432,13 +432,14 @@
       "header": "Selected Highlights"
     },
     "recommended": {
-      "same-topic": "Same Topic",
+      "same-topic": "Recommanded for topic",
```
Comment (gshilin) [inline public/locales/en/common.json:435]: recomm**E**nded

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js [hunk 2]
```diff
@@ -34,11 +34,36 @@ export const renderPlaylistUnit = (unit, t, views = -1, watchingNow = -1, sugges
     <Table.Body>
       <Table.Row verticalAlign="middle">
         <Table.Cell textAlign="left" width={4}>
-          <UnitLogo
-            unitId={unit.id}
-            collectionId={getCollectionId(unit)}
-            fallbackImg='programs'
-          />
+          <div className="recommend-cell">
```
Comment (gshilin) [inline src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:37]: Can we move this class to the above Table.Cell?

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js [hunk 3]
```diff
@@ -34,11 +34,36 @@ export const renderPlaylistUnit = (unit, t, views = -1, watchingNow = -1, sugges
...
+            <UnitLogo
+              unitId={unit.id}
+              collectionId={getCollectionId(unit)}
+              fallbackImg='programs'
+            />
+            {(views !== -1 || watchingNow !== -1 || suggesterLabel) &&
```
Comment (gshilin) [inline src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:43]: I believe this and the following conditions are correct, but that's not clear at all. Can you please do one of the following:

- move them to constants with clear names
- make html after condition to be a function with clear name

I'd prefer both, but that's up to you

Comment (gshilin) [reply src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:43]: I understand that you just moved this block, but as you still understand what's going on here, can you make future life easier...

File: src/components/Pages/Unit/widgets/Recommended/Main/Recommended.js [hunk 4]
```diff
@@ -5,47 +5,93 @@ import { withNamespaces } from 'react-i18next';
...
+  const [unitCollections, setUnitCollections] = useState([]);
   const prevUnitId = usePrevious(unitId);
 
   const activeVariant = (abTesting && abTesting.getVariant(AB_RECOMMEND_EXPERIMENT)) || '';
 
   const wip = useSelector(state => selectors.getWip(state.recommended));
   const err = useSelector(state => selectors.getError(state.recommended));
+  const getTagById = useSelector(state => tagsSelectors.getTagById(state.tags));
+
+  console.log(unit);
```
Comment (gshilin) [inline src/components/Pages/Unit/widgets/Recommended/Main/Recommended.js:68]: Do you need this in prod?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js (hunk 2 above)
Comment (bbfsdev) [reply src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js:37]: No as the cell height might be much higher than the div.

File: src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js (hunk 3 above)
Comment (bbfsdev) [reply src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js]: Done.

File: src/components/Pages/Unit/widgets/Recommended/Main/Recommended.js (hunk 4 above)
Comment (bbfsdev) [reply src/components/Pages/Unit/widgets/Recommended/Main/Recommended.js]: No, removed. Nice catch.

**Comment by bbfsdev** [comment]
Done.

**Review by gshilin — APPROVED** [review]

**Pull request merged**